README.md

```diff
@@ -92,62 +95,9 @@ Congrats, you're ready to deploy an application to Optimistic Ethereum!
 It really is that easy.
 
 You can verify that everything went well by checking the `artifacts` folder that should be generated whenever you run `yarn compile`.
-Alongside the normal compiler output located at `artifacts/contracts/ERC20.sol/ERC20.json`, you should also see `artifacts/contracts/ERC20.sol/ERC20.ovm.json`.
-Here, `.ovm.json` signifies that this file has been compiled for the OVM, the **O**ptimistic **V**irtual **M**achine, as opposed to the Ethereum Virtual Machine.
-
-### Testing (Again)
-We provided you with an ERC20 test file earlier in this tutorial.
-Now it's time to test this ERC20 again.
-This time, however, we'll be testing our new OVM-compatible smart contract on top of Optimistic Ethereum.
-Luckily, this is almost as easy as compiling the contract!
-
-First, make a copy of [`optimism-tutorial/test/erc20.spec.ts`](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/test/erc20.spec.ts).
-You can name the copy whatever you'd like, perhaps `optimistic-erc20.spec.ts`.
-We'll modify this copy in just a minute.
-
-Now we're going to add another Hardhat plugin to [`optimism-tutorial/hardhat.config.ts`](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/hardhat.config.ts):
-
-```ts
-// hardhat.config.ts
-
-import '@eth-optimism/plugins/hardhat/compiler' // You already had this one.
-import '@eth-optimism/plugins/hardhat/ethers'   // Now just add this one!
-```
-
-This plugin adds a new modified version of `ethers` to Hardhat that makes it possible to test the Layer 2 version of your contracts.
-
-Finally, we're going to modify `optimistic-erc20.spec.ts` (or whatever you named your copy of the original test file).
-Don't worry though, we only have to change a single line of code to make everything work!
-Find the line of code that looks like this:
-
-```ts
-// optimistic-erc20.spec.ts
-
-import { ethers } from 'hardhat'
-```
+Alongside the normal compiler output located at `artifacts/contracts/ERC20.sol/ERC20.json`, you should also see `artifacts/contracts/ERC20.sol/ERC20-ovm.json` (or `artifacts/contracts/ERC20.sol/ERC20.ovm.json` if you're using an older version of `@eth-optimism/plugins`).
```

Comment:
I would remove any references to the old version of `@eth-optimism/plugins`. Less confusing to assume that everyone will be on the latest version + instead put a note at the top of the tutorial to make sure that people are running the correct version.